lib/item-match-numerator.js

```diff
@@ -3,6 +3,16 @@ const util = require('./util')
 const addNumItemsMatched = (resp, request, electronicResourcesAlreadyRemoved = false) => {
   if (util.checkForNestedHitsAndSource(resp, 'items')) {
     const theBib = resp.hits.hits[0]
+    // populate electronic items count from items array if necessary
+    let fallbackEResources = []
+    if (theBib.inner_hits && theBib.inner_hits.electronicResources) {
+      fallbackEResources = theBib.inner_hits.electronicResources.hits.hits[0].electronicLocator
+    } else if (theBib._source.items) {
+      fallbackEResources = theBib._source.items.find((i) => i.electronicLocator).electronicLocator
+    } else if (theBib._inner_hits && theBib.inner_hits.items) {
+      fallbackEResources = theBib._inner_hits.items.hits.hits.find((i) => i.electronicLocator).electronicLocator
+    }
+    theBib._source.numElectronicResources = theBib._source.numElectronicResources || fallbackEResources.length
```

Comment:
Won't this only operate on the first hit?

Reply:
Proposing we close this PR because it only applies to the first hit.

Reply:
Yep, closing this PR.